eventsource/eventsource-request-cancellation.any.js

```diff
@@ -1,32 +1,21 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>EventSource: request cancellation</title>
-    <script src="/resources/testharness.js"></script>
-    <script src="/resources/testharnessreport.js"></script>
-  </head>
-  <body>
-    <div id="log"></div>
-    <script>
+// META: title=EventSource: request cancellation
+
       var t = async_test();
       onload = t.step_func(function() {
         var url = "resources/message.py?sleep=1000&message=" + encodeURIComponent("retry:1000\ndata:abc\n\n");
         var es = new EventSource(url);
         es.onerror = t.step_func(function() {
           assert_equals(es.readyState, EventSource.CLOSED)
-          setTimeout(t.step_func(function () {
+          this.step_timeout(function () {
```

Comment:
I think `t.step_timeout` here and below would be clearer. At least it's not clear to me that `this` is equivalent to `t`.